e2e/framework/testserver.go

```diff
@@ -452,14 +448,12 @@ func (t *TestServer) SwitchIBFTType(typ fork.IBFTType, from uint64, to, deployme
 	}
 
 	// Start the server
-	t.cmd = exec.Command(binaryName, args...)
+	t.cmd = exec.Command(resolveBinary(), args...) //nolint:gosec
 	t.cmd.Dir = t.Config.RootDir
 
-	if t.Config.ShowsLog {
-		stdout := io.Writer(os.Stdout)
-		t.cmd.Stdout = stdout
-		t.cmd.Stderr = stdout
-	}
+	stdout := t.GetStdout()
+	t.cmd.Stdout = stdout
+	t.cmd.Stderr = stdout
 
 	return t.cmd.Run()
```

Comment:
I have one suggestion. Right now, the logs of all e2e test are saved to the files separated by server index. It's hard to know what test the log is outputted from. Is it possible to write header including test information every time before running command in each e2e test?

Reply:
Love this idea ➕